fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/sensor/LidarSensorModule.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.app.api.sensor;
+
+import org.eclipse.mosaic.lib.spatial.PointCloud;
+
+import java.util.function.Consumer;
+
+public interface LidarSensorModule {
+
+    /**
+     * Enables this LiDAR sensor by giving the sensor range.
+     *
+     * @param range the range of the LiDAR sensor.
+     */
+    void enable(double range);
```

Comment:
For instance, with this enable call the `LidarSensorModule `is structurally closer to the `PerceptionModule `than to the `BasicSensorModule`

Reply:
I'm totally with you. I moved the methods to the `Perceptive` interface and removed `Sensible`.